service/src/main/resources/db/migration/V148__pedagogical_documents.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS pedagogical_document (
+                                                    id uuid PRIMARY KEY DEFAULT ext.uuid_generate_v1mc(),
+                                                    child_id uuid REFERENCES person(id) NOT NULL,
```

Comment:
There's no index for `child_id`, so if we need to fetch documents based on child ID, all documents in the database need to be looped one-by-one.

Something like:

```
CREATE INDEX idx$pedagogical_document_child ON pedagogical_document (child_id);
```